tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -115,6 +117,7 @@ def test_restart_does_not_change_config_yaml(self, cluster):
             assert config_yaml_content_after_restart == config_yaml_content
 
     @pytest.mark.level("minimal")
+    @pytest.mark.skip("invalid")
```

Comment:
Is this test broken?

Reply:
oops did not mean to submit this to the PR, but k8s were hanging on this at one point. it is failing on main release tests with ` ModuleNotFoundError: No module named 'tests.conftest'`

Reply:
oops did not mean to submit this to the PR, but k8s tests were hanging on this at one point so I skipped it to let the tests finish. but does look like it is failing on main release tests with ` ModuleNotFoundError: No module named 'tests.conftest'`